Update doc with the new Microsoft login url generator (#1291)
demisto/content-docs: content-repo/extra-docs/articles/microsoft-auth-guide.md

To configure a Microsoft integration that uses this authorization flow with a self-deployed Azure application:

1. Make sure the needed permissions are granted for the app registration, e.g for Microsoft Graph User: API/Permission name `Directory.AccessAsUser.All` of type `Delegated`.

> 2. Enter your client ID in the ***ID*** parameter field. 

Reviewer: ```suggestion
2. Enter your client ID in the *ID* parameter field. 
```

> 3. Enter your client secret in the ***Key*** parameter field.

Reviewer: ```suggestion
3. Enter your client secret in the *Key* parameter field.
```

> 4. Enter your tenant ID in the ***Token*** parameter field.

Reviewer: ```suggestion
4. Enter your tenant ID in the *Token* parameter field.
```

5. Save the instance.

> 6. Run the `!<integration command prefix>-generate-login-url` command in the War Room and follow the instructions, e.g. for Microsoft Graph User: `!msgraph-user-generate-login-url`.

Reviewer: ```suggestion
6. Run the `!<integration command prefix>-generate-login-url` command in the War Room and follow the instructions,. For example, for Microsoft Graph User: `!msgraph-user-generate-login-url`.
```
